zhaquirks/inovelli/__init__.py

```diff
@@ -66,97 +69,45 @@
 # 1 - down button
 # 2 - up button
 # 3 - config button
+# 4 - aux down button
+# 5 - aux up button
+# 6 - aux config button
 
-BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3}
+BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3, 4: BUTTON_4, 5: BUTTON_5, 6: BUTTON_6}
 ON = "Up"
 OFF = "Down"
 CONFIG = "Config"
+AUX_ON = "Aux Up"
+AUX_OFF = "Aux Down"
+AUX_CONFIG = "Aux Config"
```

Comment:
I think these should follow HA standards and be "Aux up", "Aux down", "Aux config"?